src/Common/REST/Generated/GeneratedRestClient.cs

```diff
@@ -0,0 +1,319 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.SignalR
+{
+    using Microsoft.Rest;
+    using Microsoft.Rest.Serialization;
+    using Newtonsoft.Json;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Net;
+    using System.Net.Http;
+
+    public partial class GeneratedRestClient : ServiceClient<GeneratedRestClient>, IGeneratedRestClient
```

Comment:
is the auto-generated class naming customizable?

Reply:
yes